spec/helpers/submissions_helper_spec.rb

```diff
@@ -206,4 +206,21 @@
       expect(helper.reviewer_byline(submission)).to eq 'Rejected by '
     end
   end
+
+  context 'artist_supply_priority' do
+    context 'when is_p1' do
+      subject { helper.artist_supply_priority(is_p1: true) }
+      it { is_expected.to eq 'P1' }
+    end
+
+    context 'when target_supply' do
+      subject { helper.artist_supply_priority(is_p1: true) }
```

Comment:
This should be a `target_supply` / `p2` test I think?

Reply:
Yes :) Fixed.